src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Printing the logo and greeting message
+        String logo = "  _________   _____      _____   \n" +
+                " /   _____/  /  _  \\    /     \\  \n" +
+                " \\_____  \\  /  /_\\  \\  /  \\ /  \\ \n" +
+                " /        \\/    |    \\/    Y    \\\n" +
+                "/_______  /\\____|__  /\\____|__  /\n" +
+                "        \\/         \\/         \\/ ";
+        System.out.println("Hello! I'm SAM\n" + logo + "\n" + "What can I do for you?\n");
+
+        // Initializing scanner to read user input
+        Scanner in = new Scanner(System.in);
```

Comment:
This is great! I think you can maybe benefit from using BufferedReader instead of Scanner because Scanner tends to be very slow in practice.